src/sass/shared/variables.scss

```diff
@@ -0,0 +1,31 @@
+// Colors
+$color-dark-blue: #33526c;
+$color-light-blue: #1b78b7;
+$color-pale-blue: #dae4e4;
+$color-dark-grey: #adb6bf;
+$color-medium-grey: #e6e9ec;
+$color-pale-grey: #ecf0f1;
+$color-white: white;
+
+$color-links: $color-light-blue;
+$color-links-hover: $color-dark-blue;
+$color-links-darkbg: $color-white;
+$color-links-darkbg-hover: $color-pale-grey;
+
+// Text sizes
+$font-size-body: 20px;
+$font-size-menu: 24px;
+$font-size-submenu: 21px;
+$font-size-card-titles: 21px;
+$font-size-h1: 48px;
+$font-size-h2: 30px;
+$font-size-h3: 24px;
```

Comment:
what if we want to use the h1 size over an h3?
I'd define the vars using only semantics like:
```
$font-size-title-L: 48px;
$font-size-title-M: 30px;
$font-size-title-S: 24px;
```